bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/constants/RuntimeConstants.java

```diff
@@ -49,8 +49,6 @@ public class RuntimeConstants {
     public static final String VERSION_SEPARATOR = ":";
     public static final String UNDERSCORE = "_";
     public static final String COLON = ":";
-    public static final char DOLLAR = '$';
```

Comment:
Is it OK to remove this? Given that it's part of the runtime API?

Reply:
We checked that it's not used in other places than the language.